packages/stark-ui/src/modules/breadcrumb/components/breadcrumb.component.scss

```diff
@@ -0,0 +1,52 @@
+/* ============================================================================== */
+/*  S t a r k   B r e a d c r u m b                                               */
+/* ============================================================================== */
+/* stark: src/ui/components/breadcrumb/breadcrumb.component.pcss */
+//.stark-breadcrumb {
+//  display: none;
+//}
+
+.stark-breadcrumb {
+  display: block;
+  a {
+    cursor: pointer;
+  }
+}
+
+@media #{$tablet-only-query} {
+  .stark-breadcrumb {
+    display: block;
+    padding-left: 8px;
+    font-size: 15px;
+    line-height: 24px;
+    margin-bottom: 0;
+    color: #fff;
+    a {
+      display: inline-block;
+      padding: 0 6px;
+      color: var(--tertiary-dark-text-color);
+      font-style: italic;
+      text-decoration: none;
+      font-size: 12px;
+      &:hover {
+        color: var(--mat-primary);
+      }
+    }
+    span:first-child a {
+      padding-left: 0;
+    }
+  }
+}
+
+@media #{$desktop-query} {
+  .stark-breadcrumb {
+    margin-bottom: 20px;
+    font-size: 18px;
+    a {
+      font-size: 13px;
+      padding: 0 10px;
+    }
+  }
+}
+
+/* END stark: src/ui/components/breadcrumb/breadcrumb.component.pcss */
```

Comment:
Path is incorrect. Could you please change it by : 
`/* END stark-ui: src/modules/breadcrumb/components/breadcrumb/breadcrumb.component.scss */`